app/components/inputs/search-box.jsx

```
@@ -31,11 +32,11 @@ export default function SearchBox(props) {
   };
 
   return (
-    <div className={`search-box sm:grow sm:self-stretch sm:justify-stretch flex search-box rounded-full h-fit sm:h-14 sm:max-w-[720px] bg-[--md-sys-color-surface-container] text-[--md-sys-color-on-surface-container] ${props.className}`} role='search' onClick={()=>{if (pathname !== "/search") router.push("/search")}}>
-      <div className="state-layer sm:self-stretch relative rounded-full sm:pl-6 sm:pr-1 sm:gap-1 flex items-center sm:w-full">
+    <div className={twMerge('search-box', props.className, "grow self-stretch justify-stretch h-full min-h-14 max-w-[720px]", 'bg-[--md-sys-color-surface-container] text-[--md-sys-color-on-surface-container] rounded-full')} role='search' onClick={()=>{if (pathname !== "/search") router.push("/search")}}>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)